dotcom-rendering/src/web/components/Island.tsx

```diff
@@ -20,13 +20,21 @@ interface ClientOnlyProps {
 	expediteLoading?: boolean;
 }
 
+interface InteractionProps {
+	deferUntil: 'interaction';
+	clientOnly?: false;
+	placeholderHeight?: never;
+	children: JSX.Element;
+	expediteLoading?: false;
```

Comment:
```suggestion
	expediteLoading?: never;
```

Does this work with never? If so I guess that's a better dev ex.